port/action/schema.go

```diff
@@ -1111,6 +1153,10 @@ resource "port_action" "create_microservice" {
                             }]
                         })
 					}
+                    sort = {
```

Comment:
Is it me or is the indent here a bit askew?

Reply:
check how it looks locally